tests/conftest.py

```diff
@@ -117,8 +124,12 @@ def _config(app, **kwargs):
 
 
 @pytest.fixture
-def use_flask_limiter(app):
-    """Activate flask limiter."""
+def use_flask_limiter():
```

Comment:
Since now you have an application fixture with no Limiter this fixture seems redundant since instead of "enabling"  and "disabling" the limiter you can pass to the tests that use it or not the appropriate fixture.

Reply:
true!